src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedEntryPointSymbol.cs

```diff
@@ -35,23 +35,7 @@ internal static SynthesizedEntryPointSymbol Create(SynthesizedInteractiveInitial
             }
             else
             {
-                var taskType = compilation.GetWellKnownType(WellKnownType.System_Threading_Tasks_Task);
-#if DEBUG
-                HashSet<DiagnosticInfo> useSiteDiagnostics = null;
-                Debug.Assert(taskType.IsErrorType() || initializerMethod.ReturnType.IsDerivedFrom(taskType, TypeCompareKind.IgnoreDynamicAndTupleNames, useSiteDiagnostics: ref useSiteDiagnostics));
-#endif
-                ReportUseSiteDiagnostics(taskType, diagnostics);
-                var getAwaiterMethod = taskType.IsErrorType() ?
-                    null :
-                    GetRequiredMethod(taskType, WellKnownMemberNames.GetAwaiter, diagnostics);
-                var getResultMethod = ((object)getAwaiterMethod == null) ?
-                    null :
-                    GetRequiredMethod(getAwaiterMethod.ReturnType, WellKnownMemberNames.GetResult, diagnostics);
-                return new ScriptEntryPoint(
-                    containingType,
-                    compilation.GetSpecialType(SpecialType.System_Void),
-                    getAwaiterMethod,
-                    getResultMethod);
+                return new ScriptEntryPoint(containingType, compilation.GetSpecialType(SpecialType.System_Void));
```

Comment:
>compilation.GetSpecialType(SpecialType.System_Void) [](start = 60, length = 51)

As I mentioned before, we should never assume that all types are available and good to use. I think we have a helper in the Binder that does all appropriate error reporting, use it instead. Also, please add test for a situation when void is missing.